circleci: failed tests: TestErrcheck TestGoimports TestReturnCheck

The following test appears to have failed:

[#20808](https://circleci.com/gh/cockroachdb/cockroach/20808):

```
--- PASS: TestEnvutil (0.00s)
=== RUN TestErrcheck
/go/src/github.com/cockroachdb/cockroach/storage/helpers_test.go:132:3: undeclared name: atomic
/go/src/github.com/cockroachdb/cockroach/storage/helpers_test.go:134:3: undeclared name: atomic
error: failed to check packages: could not type check: couldn't load packages due to errors: github.com/cockroachdb/cockroach/storage
--- FAIL: TestErrcheck (0.00s)

=== RUN TestForbiddenImports
--- PASS: TestForbiddenImports (0.00s)
=== RUN TestGoSimple
--- PASS: TestGoSimple (0.00s)
=== RUN TestGofmtSimplify
--- PASS: TestGofmtSimplify (0.00s)
=== RUN TestGoimports
--- FAIL: TestGoimports (0.00s)
storage/helpers_test.go
=== RUN TestGolint
--- PASS: TestGolint (0.00s)
=== RUN TestGrpc
--- PASS: TestGrpc (0.00s)
=== RUN TestImportNames
--- PASS: TestImportNames (0.00s)
=== RUN TestIneffassign
--- PASS: TestIneffassign (0.00s)
=== RUN TestMissingLeakTest
--
=== RUN TestProtoMarshal
--- PASS: TestProtoMarshal (0.00s)
=== RUN TestReturnCheck
/go/src/github.com/cockroachdb/cockroach/storage/helpers_test.go:132:3: undeclared name: atomic
/go/src/github.com/cockroachdb/cockroach/storage/helpers_test.go:134:3: undeclared name: atomic
--- FAIL: TestReturnCheck (0.00s)

=== RUN TestSafeSQL
--- PASS: TestSafeSQL (0.00s)
=== RUN TestSyncMutex
--- PASS: TestSyncMutex (0.00s)
=== RUN TestTabsInShellScripts
--- PASS: TestTabsInShellScripts (0.00s)
=== RUN TestTimeutil
--- PASS: TestTimeutil (0.00s)
=== RUN TestUnconvert
--
/go/src/github.com/cockroachdb/cockroach/storage/helpers_test.go:134:3: undeclared name: atomic
2016/08/01 20:27:56 could not type check: couldn't load packages due to errors: github.com/cockroachdb/cockroach/storage
--- PASS: TestVarcheck (0.00s)
=== RUN TestVet
--- PASS: TestVet (0.00s)
FAIL check-style 0.000s
Makefile:189: recipe for target 'check' failed
make: *** [check] Error 1

```
Please assign, take a look and update the issue accordingly.

--- a/storage/helpers_test.go
+++ b/storage/helpers_test.go
@@ -125,14 +125,10 @@ func (s *Store) SetSplitQueueActive(active bool) {
 	s.setSplitQueueActive(active)
 }
 
-// SetReplicaScannerDisabled turns replica scanning off or on as directed. Note
-// that while disabled, removals are still processed.
-func (s *Store) SetReplicaScannerDisabled(disabled bool) {
-	if disabled {
-		atomic.StoreInt32(&s.scanner.disabled, 1)
-	} else {
-		atomic.StoreInt32(&s.scanner.disabled, 0)
-	}
+// SetReplicaScannerActive enables or disables the scanner. Note that while
+// inactive, removals are still processed.
+func (s *Store) SetReplicaScannerActive(active bool) {
+	s.setScannerActive(active)
 }
 
 // GetLastIndex is the same function as LastIndex but it does not require

--- a/storage/raft_log_queue_test.go
+++ b/storage/raft_log_queue_test.go
@@ -204,7 +204,7 @@ func TestProactiveRaftLogTruncate(t *testing.T) {
 	store, _, stopper := createTestStore(t)
 	defer stopper.Stop()
 
-	store.SetReplicaScannerDisabled(true)
+	store.SetReplicaScannerActive(false)
 
 	r, err := store.GetReplica(1)
 	if err != nil {

--- a/storage/replicate_test.go
+++ b/storage/replicate_test.go
@@ -32,7 +32,7 @@ func TestEagerReplication(t *testing.T) {
 
 	// Disable the replica scanner so that we rely on the eager replication code
 	// path that occurs after splits.
-	store.SetReplicaScannerDisabled(true)
+	store.SetReplicaScannerActive(false)
 
 	if err := server.WaitForInitialSplits(store.DB()); err != nil {
 		t.Fatal(err)

